frontend/javascripts/oxalis/view/statusbar.js

```diff
@@ -0,0 +1,220 @@
+// @flow
+import { Space, Tooltip } from "antd";
+import _ from "lodash";
+import { connect } from "react-redux";
+import React from "react";
+
+import type { OxalisState } from "oxalis/store";
+import {
+  type Vector2,
+  type Vector3,
+  type OrthoView,
+  OrthoViews,
+  type VolumeTool,
+  VolumeToolEnum,
+} from "oxalis/constants";
+import { getCurrentResolution } from "oxalis/model/accessors/flycam_accessor";
+import api from "oxalis/api/internal_api";
+import { calculateGlobalPos } from "oxalis/controller/viewmodes/plane_controller";
+import Cube from "oxalis/model/bucket_data_handling/data_cube";
+import { V3 } from "libs/mjs";
+import Model from "oxalis/model";
+import { MoreOutlined } from "@ant-design/icons";
+
+type OwnProps = {||};
+type StateProps = {|
+  activeResolution: Vector3,
+  activeViewport: OrthoView,
+  mousePosition: ?Vector2,
+  isSkeletonAnnotation: boolean,
+  activeTool: ?VolumeTool,
+|};
+type Props = {| ...OwnProps, ...StateProps |};
+type State = {||};
+
+const borderToggleButtonMargin = 40;
+const fontSize = 14;
+const spaceBetweenItems = 25;
+const lineColor = "rgba(255, 255, 255, 0.67)";
+
+const statusbarStyle: Object = {
+  marginLeft: borderToggleButtonMargin,
+  marginRight: borderToggleButtonMargin,
+  fontSize,
+  display: "flex",
+  flexWrap: "wrap",
+  overflow: "hidden",
+};
+const defaultIconStyle = { height: fontSize - 1, marginTop: 2 };
```

Comment:
Why `fontSize - 1`? At least on my screen, this makes the icons quite blurry. For example, the "movement" dashes next to the left-click-drag-icon becomes barely visible. When I bumped the height via the devtools, the icon became clear again and the height also looked alright. Or do you see any downsides to this?